CHANGELOG.md

```diff
@@ -2,15 +2,19 @@ Changelog for the bc-lightstep-ruby gem.
 
 h3. Pending Release
 
-h3. 1.3.1
+h3. 1.3.3
```

Comment:
This should just be under `Pending Release`, as we don't know yet whether this will go under 1.3.3 or 1.4.0